ee/fleetd-chrome/updates-beta.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<gupdate xmlns='http://www.google.com/update2/response' protocol='2.0'>
+  <app appid='bfleegjcoffelppfmadimianphbcdjkb'>
+    <updatecheck codebase='https://chrome-beta.fleetdm.com/fleetd.crx' version='1.1.0' />
```

Comment:
Other than the Beta appid being different from the operational appid, wouldn't it be better to also mark the version with the letter `b` to avoid confusion?

i.e. version='b1.1.0'

Reply:
Or `1.1.0+beta`

Reply:
IIRC Chrome is very picky about what versions it will update to and I could not find any documentation on acceptable strings. Adding a `-beta` or `+beta` suffix might work but it would need testing.

Reply:
NP. 
We can start as-is and check for this later as an improvement.